pkgs/development/ocaml-modules/ocaml-libvirt/default.nix

```diff
@@ -1,20 +1,26 @@
-{ stdenv, fetchurl, libvirt, ocaml, findlib }:
+{ stdenv, fetchgit, libvirt, autoconf, ocaml, findlib }:
 
 stdenv.mkDerivation rec {
   name = "ocaml-libvirt-${version}";
-  version = "0.6.1.4";
+  version = "0.6.1.5"; # FIXME: libguestfs depends on not yet released 0.6.1.5, so the latest post-0.6.1.4 master from git is used
```

Comment:
```libguestfs``` and ```ocaml-libvirt``` are made by the same author, @rwmjones, so he is dobtfully restricted by not-officially-releasing the library.
0.6.1.4 was released in August 2014.